includes/paypro/wc/gateways/abstract.php

```diff
@@ -187,6 +294,13 @@ public function getIconUrl() {
         return PAYPRO_WC_PLUGIN_URL . 'assets/images/' . $this->id . '.png';
     }
 
+    /**
+     * Check if the gateway supports refunds.
+     */
+    public function supportsRefunds() {
+        return $this->supports_refunds;
```

Comment:
Can the gateway's `supports` array be used to check if refunds are available?

Reply:
I refactored the code a bit, but did it the other way around. This will make more sense especially with the subscriptions support.